sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaReadSchemaTransformProvider.java

```diff
@@ -103,6 +105,16 @@ public Row apply(byte[] input) {
     };
   }
 
+  public static SerializableFunction<KV<byte[], byte[]>, Row> getRawBytesKvToRowFunction(
+      Schema rawSchema) {
+    return new SimpleFunction<KV<byte[], byte[]>, Row>() {
+      @Override
+      public Row apply(KV<byte[], byte[]> input) {
+        return Row.withSchema(rawSchema).addValues(input.getKey(), input.getValue()).build();
+      }
+    };
+  }
+
```

Comment:
This should probably be a fixed function with a fixed output Schema (i.e. not a method)